<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTeacherManger</class>
 <widget class="QMainWindow" name="CTeacherManger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>All teacher</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MangerTeacher</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NoCounselorclass</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Don't have teacher's Course</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="6,0">
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_AppointCoun">
          <property name="text">
           <string>任命导员</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_CancelCoun">
          <property name="text">
           <string>卸任导员</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_AppointCour">
          <property name="text">
           <string>委派课程</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_CancelCour">
          <property name="text">
           <string>卸任课程</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_AddTeacher">
          <property name="text">
           <string>添加教师</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_DelTeacher">
          <property name="text">
           <string>删除教师</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
